sections/shared/Layout/AppLayout/Header/BalanceActions/BalanceActions.tsx

```diff
@@ -0,0 +1,191 @@
+import { FlexDivRow, FlexDivRowCentered } from 'styles/common';
+import { formatCurrency, zeroBN } from 'utils/formatters/number';
+import { getDisplayAsset, getMarketKey } from 'utils/futures';
+
+type ReactSelectOptionProps = {
+	label: string;
+	synthIcon: string;
+	marketRemainingMargin?: string;
+	onClick?: () => {};
+};
+
+type FuturesPositionTableProps = {
+	futuresMarkets: FuturesMarket[];
+	futuresPositions: FuturesPosition[];
+	setShowUniswapWidget: Dispatch<SetStateAction<boolean>>;
+};
+
+const BalanceActions: FC<FuturesPositionTableProps> = ({
+	futuresMarkets,
+	futuresPositions,
+	setShowUniswapWidget,
+}) => {
+	const { t } = useTranslation();
+	const theme = useTheme();
+	const router = useRouter();
+	const { network } = Connector.useContainer();
+
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { useSynthsBalancesQuery } = useSynthetixQueries();
+	const synthsBalancesQuery = useSynthsBalancesQuery(walletAddress);
+	const sUSDBalance = synthsBalancesQuery?.data?.balancesMap?.[Synths.sUSD]?.balance ?? zeroBN;
+
+	const balanceLabel = formatCurrency(Synths.sUSD, sUSDBalance, { sign: '$' });
+
+	const totalRemainingMargin = futuresPositions.reduce(
+		(prev, position) => prev.add(position.remainingMargin),
+		zeroBN
+	);
+
+	const setMarketConfig = (asset: string) => {
+		const remainingMargin =
+			futuresPositions.find((posittion) => posittion.asset === asset)?.remainingMargin ?? zeroBN;
+
+		const marketKey = getMarketKey(asset, network.id);
+
+		return {
+			label: `${getDisplayAsset(asset, network.id)}-PERP`,
+			synthIcon: marketKey,
+			marketRemainingMargin: formatCurrency(Synths.sUSD, remainingMargin, { sign: '$' }),
+			onClick: () => router.push(`/market/${marketKey}`),
+		};
+	};
+
+	const OPTIONS = [
+		{
+			label: 'header.balance.margin-label',
+			totalAvailableMargin: formatCurrency(Synths.sUSD, totalRemainingMargin, { sign: '$' }),
+			options: futuresMarkets.map((market) => setMarketConfig(market.asset)),
+		},
+	];
+
+	const OptionsGroupLabel: FC<{
+		label: string;
+		totalAvailableMargin?: string;
+	}> = ({ label, totalAvailableMargin }) => {
+		return (
+			<FlexDivRow>
+				<Container>{t(label)}</Container>
+				<Container>{totalAvailableMargin}</Container>
+			</FlexDivRow>
+		);
+	};
+
+	const formatOptionLabel = ({
+		label,
+		synthIcon,
+		marketRemainingMargin,
+		onClick,
+	}: ReactSelectOptionProps) => (
+		<LabelContainer noPadding={synthIcon === 'sUSD'} onClick={onClick}>
+			<FlexDivRow>
+				{synthIcon && <StyledCurrencyIcon currencyKey={synthIcon} />}
+				<StyledLabel noPadding={!synthIcon}>{t(label)}</StyledLabel>
+			</FlexDivRow>
+			<Container>{marketRemainingMargin}</Container>
+		</LabelContainer>
+	);
+
+	const Group: FC<GroupProps<any>> = ({ children, ...props }) => (
+		<components.Group {...props}>
+			<StyledOptions>{children}</StyledOptions>
+			<StyledButton onClick={() => setShowUniswapWidget(true)}>
+				{t('header.balance.get-more-susd')}
+			</StyledButton>
```

Comment:
Same here, make sure `text-transform` does not automatically `capitalize` the `sUSD`